templates/repo/issue/view_content/comments.tmpl

```diff
@@ -295,7 +295,9 @@
 				{{template "shared/user/avatarlink" dict "Context" $.Context "user" .Poster}}
 				<span class="text grey muted-links">
 					{{template "shared/user/authorlink" .Poster}}
-					{{$.locale.Tr "repo.issues.due_date_added" .Content $createdStr | Safe}}
+					{{$.locale.Tr "repo.issues.due_date_added"}}
+					{{template "shared/datetime/short" (dict "Datetime" .Content "Fallback" .Content)}}
+					{{$createdStr}}
```

Comment:
I can only see one suitable solution that will take some refactoring:
Add a method `Comment.AsTimeStamp()` that only works on comments with the correct comment types where the content of the comment is a timestamp in string form.
Then, you can call `.AsTimeStamp().TimeSinceUnix()`.
You may even decide to panic in this new method instead of throwing an error as it can only happen when someone calls this method in the wrong place.

Reply:
It could be like this:

```
{{$s = printf "<relative-time ...>" ....}}
```

Then pass the `$s` to `Tr`